app/views/posts/index.html.erb

```diff
@@ -1 +1,21 @@
-<h1>All posts</h1>
+<section class="mt-5">
+  <div class="container">
+    <%= render partial: "users/user-header", locals: {user: @user} %>
+  <div class="row border d-block m-0 mt-3">
+    <% @user.recent_posts.each do |post| %>
+    <%= link_to user_post_path(user_id: @user.id, id: post.id) do %>
+    <%= render partial: "post-preview", locals: {post: post} %>
+  <div class="border m-2">
+    <% if post.comments.length == 0 %>
+      <span class="">No comments</span>
+    <% else %>
+      <% post.recent_comments.each do |comment| %>
+        <p class="m-2 border-bottom"><%= comment.author.name %>: <%= comment.text %></p>
+      <% end %>
+      <% end %>
+    </div>
+    <% end %>
+    <% end %>
+  </div>
+</div>
+</section>
```

Comment:
- [ ] On the post index page, each post should have the number of comments and likes under it as seen in the wireframe. Kindly add it.

<img width="274" alt="chrome_sU3KuUIl63" src="https://user-images.githubusercontent.com/81468675/163546537-7e07a623-8527-4b8f-ac47-b2bf2a9bd5e4.png">